src/js/components/Header/stories/Sticky.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import {
+  Anchor,
+  Avatar,
+  Card,
+  Grid,
+  Header,
+  Nav,
+  Menu,
+  Page,
+  PageContent,
+  Paragraph,
+} from 'grommet';
+import { Power, User } from 'grommet-icons';
+
+export const Sticky = () => (
+  // Uncomment <Grommet> lines when using outside of storybook
+  // <Grommet theme={...}>
+  <Page kind="narrow">
+    <Header direction="column" sticky="scrollup" background="light-4">
```

Comment:
This feels awkward to me. I'm also not sure it's best practice to have nested `<header>` tags in the DOM. It would make more sense more me if line22 with the second header was just implemented as a box with appropriate props.

Reply:
good point that would require us coping most all of the `props` from Header to a Box so not sure about that either but would make more sense than having 2 `Headers` in the dom but also more of a chance for users to forget and not get the right behavior

Reply:
It's tough, because we want half of the Header containing the PageContent and half of it within the PageContent. Some other options:
- make Header smarter such that includes it's own PageContent via something like `<Header page />`?
- Use `as={Box}` for the inner Header
- ???

Reply:
adding `as={Box}` would be a lot easier then adding all of the other styles

Reply:
> * make Header smarter such that includes it's own PageContent via something like `<Header page />`?

I had been thinking something along these lines ☝️ , where Header would consume PageContext in the same manner that PageContent does.